### PR TITLE
SAK-46486 Profile: Improve Accessibility for Edit Buttons

### PR DESCRIPTION
**Pull request by V100X**
Objectives for _My profile_ section

- Make button visibility consistent: After saving or canceling changes, it wasn't visible when focused using the tab key.
- Enhance the way info is provided to secreen reader.

Check [SAK-46486](https://sakaiproject.atlassian.net/browse/SAK-46486) for more details.

I've also formatted the code, so I'll highlight the relevant changes.

### CODE REVIEW

**Review by ottenhoff — APPROVED** [review]
Nice job!

File: library/src/morpheus-master/sass/modules/tool/profile2/_profile2.scss [hunk 1]
```diff
@@ -18,25 +18,35 @@
...
-	.edit-image-button {
+	.profileContent .radio label, .profileContent .checkbox label {
+		font-size: 1.2em;
+	}
+
+	.editable:hover > .edit-button, .edit-button:focus {
 		@include sakai_secondary_button();
-		position:absolute;
-		right:0;
-		top:0;
```
Comment (V100X) [inline library/src/morpheus-master/sass/modules/tool/profile2/_profile2.scss:39]: Button visibility management now is handled by SCSS.

File: profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyBusinessDisplay.java [hunk 2]
```diff
@@ -152,12 +145,10 @@ public void onClick(AjaxRequestTarget target) {
...
 					target.appendJavaScript("setMainFrameHeight(window.name);");
 				}
-
 			}
-
 		};
-		editButton.add(new Label("editButtonLabel", new ResourceModel(
-				"button.edit")));
+		editButton.add(new Label("editButtonLabel", new ResourceModel("button.edit")));
+		editButton.add(new AttributeModifier("aria-label", new ResourceModel("accessibility.edit.business")));
```
Comment (V100X) [inline profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyBusinessDisplay.java:151]: This is the way `aria-label` is added to the button. Applied to every Java file, as well as the needed import.

**Comment by V100X** [comment]
Btw, should I update the License info on top of each file?

**Comment by ottenhoff** [comment]
I wouldn't bother with the license

**Pull request merged**